pkg/webhooks/policymutation.go

```diff
@@ -227,6 +265,10 @@ type kyvernoRule struct {
 }
 
 func generateRuleForControllers(rule kyverno.Rule, controllers string) kyvernoRule {
+	if strings.HasPrefix(rule.Name, "autogen-") {
+		return kyvernoRule{}
+	}
```

Comment:
If a Pod's rule changes on an UPDATE, should the auto-gen rule change as well? Kind of "sync" the changes?

Reply:
Yes, the current changes ensure that if a clusterpolicy rule is edited, the autogen rule will also be edited. You can test this out yourself

Reply:
Can you point me to the code? I must have missed something.